eox_nelp/stats/api/v1/views.py

```diff
@@ -0,0 +1,68 @@
+"""Stats API v1 view file.
+
+views:
+    GeneralTenantStatsView: View that handles the general tenant stats.
+"""
+from django.contrib.auth import get_user_model
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from eox_nelp.stats import metrics
+
+User = get_user_model()
+
+
+class GeneralTenantStatsView(APIView):
+    """Class view. Handle general tenant stats.
+
+    ## Usage
+    The components key depends on the setting ALLOWED_VERTICAL_BLOCK_TYPES, this should be
+    a list of strings like the following
+
+    ``` json
+    [" problem", "video", "discussion"]
+    ```
+
+    ### **GET** /eox-nelp/api/stats/v1/tenant-stats/
+
+    **GET Response Values**
+    ``` json
+    {
+        "learners": 1,
+        "instructors": 1,
+        "courses": {
+            "total_courses": 4,
+            "metrics": [
+                {
+                    "id": "course-v1:patata+CS102+2023",
+                    "name": "PROCEDURAL SEDATION AND ANALGESIA COURSE",
+                    "learners": 0,
+                    "instructors": 1,
+                    "sections": 18,
+                    "sub_sections": 144,
+                    "units": 184,
+                    "components": {
+                        "discussion": 0,
+                        "drag-and-drop-v2": 0,
+                        "html": 133,
+                        "openassessment": 0,
+                        "problem": 49,
+                        "video": 0
+                    }
+                },
+                ...
+            ]
+        }
+    }
+    ```
+    """
+
+    def get(self, request):
+        """Return general tenant stats."""
+        tenant = request.site.domain
+
+        return Response({
+            "learners": metrics.get_learners_metric(tenant),
```

Comment:
Why if I dont have the videos and problems section if they should be by tenant.
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/75d6ea34-e9f9-430c-b0ca-f4d39533704c)

Why for this keys video and problems I have to calculate them later.
https://github.com/nelc/frontend-essentials/pull/20/files#diff-683674ae0d4e56c5cc60a4c990b1dc7a78426e416462479e8bdb38f1fcff6909R105